sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/secret_client.hpp

```diff
@@ -51,7 +51,6 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Secrets {
     // Using a shared pipeline for a client to share it with LRO (like delete key)
     Azure::Core::Url m_vaultUrl;
     std::string m_apiVersion;
-    std::shared_ptr<Azure::Core::Http::_internal::HttpPipeline> m_pipeline;
```

Comment:
`#include <azure/core/internal/http/pipeline.hpp>` is probably not needed anymore

Reply:
done